### PR TITLE
feat/add deprecation warning to all embed code

### PR DESCRIPTION
**Pull request by rbiseck3**
### Description
Related PR to move the code over: https://github.com/Unstructured-IO/unstructured-ingest/pull/92

Also removed the console script that exposes ingest.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
-## 0.15.10
+## 0.15.11
 
 ### Enhancements
 
 * **Enhance `pdfminer` element cleanup** Expand removal of `pdfminer` elements to include those inside all `non-pdfminer` elements, not just `tables`.
```
Comment (MthwRobinson) [inline CHANGELOG.md:5]: Pretty sure these bullets were in the `0.15.10` release. If so, may need a fresh section for `0.15.11`

Comment (rbiseck3) [reply CHANGELOG.md:5]: Those look accurate based on the release notes: https://github.com/Unstructured-IO/unstructured/releases/tag/0.15.10

Comment (MthwRobinson) [reply CHANGELOG.md:5]: Looks like the `0.15.10` section is missing on the branch and `0.15.10` was released already. Think your new entry needs to be in a fresh `0.15.11` section.

image

Comment (rbiseck3) [reply CHANGELOG.md:12]: Fixed

**Review by MthwRobinson — APPROVED** [review]
LGTM!

**Pull request merged**